opencti-platform/opencti-front/src/components/Loader.tsx

```diff
@@ -57,11 +57,13 @@ const Loader: FunctionComponent<LoaderProps> = ({
   withTopMargin = false,
 }) => {
   const classes = useStyles();
+
+  const { settings } = useContext(UserContext);
+  const isEnterpriseEdition = isNotEmptyField(settings?.enterprise_edition);
+  const isWhitemarkEnable = !settings?.platform_whitemark || !isEnterpriseEdition;
```

Comment:
I find these variable names unclear. 
What about:
```suggestion
  const hasFiligranLoader= isNotEmptyField(settings?.enterprise_edition) || !settings?.platform_whitemark;
```
And use `hasFiligranLoader` below